### PR TITLE
Handle +/- before numbers in CSS

### LINKED ISSUE #2712: Decimal formatting doesn't get all decimals in a rule.
Current Prettier:
```shell
$ ./node_modules/prettier/bin/prettier.js -v
1.6.0
```

Create a file called test.css:
```css
$ echo ".test{margin: -.3rem .5rem .2px -.5vh;}" |> test.css
```

Run prettier:
```shell
$ ./node_modules/prettier/bin/prettier.js test.css
.test {
  margin: -.3rem 0.5rem 0.2px -0.5vh;
}
```

Notice that the first value (-.3rem) still does not have leading zero.

### PR DESCRIPTION
**Pull request by xtian**
The code to add leading zeros to decimal numbers didn't account for negative numbers. This PR fixes that.

Let me know if this solution is acceptable. Thanks! :)

Fixes #2712

### CODE REVIEW

File: src/util.js [hunk 1]
```diff
@@ -576,7 +576,7 @@ function printNumber(rawNumber) {
       // Remove unnecessary scientific notation (1e0).
       .replace(/^([\d.]+)e[+-]?0+$/, "$1")
       // Make sure numbers always start with a digit.
-      .replace(/^\./, "0.")
+      .replace(/^(\+|\-)?\./, "$10.")
```
Comment (lydell) [inline src/util.js]: This is better written as `/^([+-])?\./`

Comment (xtian) [reply src/util.js]: Yep I noticed that I was doing the scientific notation :)

**Comment by xtian** [comment]
It'd be super helpful if this fix could be released :)

**Comment by lydell** [comment]
@xtian You can track the release of 1.6.2 in #2804.

**Pull request merged**